src/main/resources/templates/codeofconduct/README.md

```diff
@@ -0,0 +1,32 @@
+# Code of Conduct Template: Adapt to your demands
```

Comment:
In case the instructor does not change anything, the word "template" should not be shown to the students.
```suggestion
<!-- Code of Conduct Template: Adapt to your demands -->
```

Reply:
Agree 0f80c151ebe75fdc2c3ab02640561b8977893adf